docs/release-notes/NuGet-4.7-RTM.md

```diff
@@ -7,16 +7,21 @@ ms.date: 5/14/2018
 ms.topic: conceptual
 ---
 
-# NuGet 4.7 RTM Release Notes
+# NuGet 4.7 Release Notes
 
 [Visual Studio 2017 15.7 RTW](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes) comes with [NuGet 4.7.0](https://dist.nuget.org/win-x86-commandline/v4.7.0/nuget.exe).
 
-## Summary: What's New in this Release
+## Summary: What's New in 4.7.0
 
 * We have augmented package signing to enable [Repository Signed packages](https://github.com/NuGet/Home/wiki/Repository-Signatures)
 
 * With Visual Studio Version 15.7, we have introduced the capability to [migrate existing projects that use the packages.config format to use PackageReference](https://docs.microsoft.com/en-us/nuget/reference/migrate-packages-config-to-package-reference) instead.
 
+## Summary: What's New in 4.7.3
+
+* Security Fix: Permissions on files created inside ~/.nuget are too open [#7673](https://github.com/NuGet/Home/issues/7673) [CVE-2019-0757](https://portal.msrc.microsoft.com/en-us/security-guidance/advisory/CVE-2019-0757)
+* Security Fix: Files inside of NUPKGs can have a relative path above the NUPKG directory [#7906](https://github.com/NuGet/Home/issues/7906)
```

Comment:
minor: permissions on files was fixed in 4.7.2. 4.7.3 contains the files inside of nupkg fix.